app/controllers/api/v1/npq_applications_controller.rb

```diff
@@ -68,6 +72,20 @@ def access_scope
       def npq_application
         @npq_application ||= npq_lead_provider.npq_applications.includes(:cohort, :npq_course, participant_identity: [:user]).find(params[:id])
       end
+
+      def funded_place
+        permitted_params.dig("attributes", "funded_place")
+      end
+
+      def permitted_params
+        params.require(:data).permit(:type, attributes: %i[schedule_identifier funded_place])
```

Comment:
could we remove `schedule_identifier` from here as it's only permitted in v3?

Reply:
removed from v1+v2 👍